examples/mqtt_handler.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: 2022 vladak
+# SPDX-License-Identifier: Unlicense
+"""
+MQTT logging handler - log records will be published as MQTT messages
+"""
+
+import adafruit_minimqtt.adafruit_minimqtt as MQTT
+
+# adafruit_logging defines log levels dynamically.
+# pylint: disable=no-name-in-module
+from adafruit_logging import NOTSET, Handler, LogRecord
+
+
+class MQTTHandler(Handler):
```

Comment:
I think some of our infrastructure tooling assumes that files in the examples folder will be prepended with the library name i.e. all files in the Logging library example folder should be `logging_[something]`

I think having this class in the file `mqtt_handler.py` may cause trouble in a few spots. I would lean toward moving this class into `logging_mqtt_handler.py`. The class isn't too much code and is used primarily by that example so having it in the same file is fine in this case imo. 

If you'd prefer to keep the class in it's own file separate from the example we'll need to tweak the name some so that both start with the `logging_` prefix. Perhaps `logging_mqtt_handler.py` and `logging_mqtt_handler_test.py`

Reply:
Thanks for taking a look ! Once you come back with your findings, I will make the changes in one go.

Reply:
I tested this successfully with CPython from PC. It's working successfully and uploading the logs to Adafruit IO. Thanks again for working on this. I think it's good to go once the file naming is sorted.

Eventually It would be great to add a circuitpython version that runs on the microcontroller to this repo as well, I intend to try to adapt your example for that and I can add it in a future PR afterward.

Reply:
I was able to successfully test this on a ESP32 S3 Feather TFT microcontroller as well, all looking good in that context too.

Reply:
Yeah, I've been using the MQTTHandler on one of my QtPy's for couple of days now.

Reply:
reshuffled the file.